### ISSUE TITLE
Dashboard is showing "srczip" dependency for Guava

### ISSUE THREAD

**Issue by suztomo** [GoogleCloudPlatform/cloud-opensource-java#1534]
The dashboard showing srczip dependency in Guava 29.0.

<img width="1276" alt="Screen Shot 2020-07-14 at 17 11 47" src="https://user-images.githubusercontent.com/28604/87476863-1ba9e580-c5f5-11ea-872b-60e94b4f0927.png">

As per https://search.maven.org/artifact/com.google.guava/guava/29.0-jre/bundle, the dependency is added as per the profile and it's system-scope:

```
  <profiles>
    <profile>
      <id>srczip-parent</id>
      <activation>
        <file>
          <exists>${java.home}/../src.zip</exists>
        </file>
      </activation>
      <dependencies>
        <dependency>
          <groupId>jdk</groupId>
          <artifactId>srczip</artifactId>
          <version>999</version>
          <scope>system</scope>
          <systemPath>${java.home}/../src.zip</systemPath>
          <optional>true</optional>
        </dependency>
      </dependencies>
    </profile>
```

**Comment by elharo** [comment]
That's just the JDK classes. I'm sure the Guava team has some reason for putting it there, but we should probably just special case it and not report it.

**Comment by suztomo** [comment]
As per https://github.com/google/guava/commit/79983a915f6c37bda832b37ab6f9e7ac09c57388, the srczip dependency helps building javadoc of Guava. Because it's not for Guava consumers, it's safe to remove srczip.

**Comment by suztomo** [comment]
> system
> This scope is similar to provided except that you have to provide the JAR which contains it explicitly. The artifact is always available and is not looked up in a repository.

https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html

Because the dependency is not through a repository. Linkage Checker cannot take any action on system-scope dependency.

**Comment by elharo** [comment]
fixed by #1539
